CONTRIBUTING.md

```diff
@@ -0,0 +1,175 @@
+# Contribution Guidelines
+
+This document contains information of use to developers looking to improve this codebase.  See [README.md](https://github.com/tvkitchen/tv-kitchen/blob/master/README.md) for an introduction to this project. All discussions about this project fall under our [code of conduct](CODE_OF_CONDUCT.md).
+
+## Submitting and Reviewing Code
+
+This project has a home on [GitHub](https://github.com/tvkitchen).  Please submit [pull requests](https://help.github.com/articles/about-pull-requests/) (PRs) there.
+
+Please submit changes via pull request, even if you have direct commit access to the repository.  The PR process allows us to get additional eyes on change proposals, and ensures that your changed code [builds cleanly via Travis CI's automated builds](https://travis-ci.org/tvkitchen/tv-kitchen/).
+
+As you work on your branch, try to test it locally to ensure that it still builds and deploys properly, and that you haven't introduced new accessibility bugs. See `docs/TESTING.md` for more instructions.
```

Comment:
May need to either put TESTING.md in the same repo or change this so it's no longer a link. Unclear whether GitHub has some magic way of handling this. Should check.